tests/testsuite.src/syn_misc.at

```diff
@@ -3530,14 +3526,62 @@ AT_DATA([prog.cob], [
 
 AT_CHECK([$COMPILE_ONLY prog.cob], [1], [],
 [prog.cob:19: error: invalid conditional expression
-prog.cob:16: error: invalid expression
+prog.cob:16: error: invalid expression: boolean expected with logical operator
 prog.cob:28: error: wrong number of WHEN parameters
 prog.cob:31: error: wrong number of WHEN parameters
 ])
 
 AT_CLEANUP
 
 
+AT_SETUP([Condition IS ZERO AND])
+AT_KEYWORDS([condition zero])
```

Comment:
no need to re-iterate title keywords, either adjust the title or comment the keywords for now